Update readme (#21)
1Password/kubernetes-secrets-injector: README.md

          - name: DB_PASSWORD
            value: op://my-vault/my-item/sql/password

>         - name: my-app #because my-app is not listed in the inject annotation above this container will not be injected with secrets

Reviewer: Not: Add an extra space for better clarity of the comment.
```suggestion
        - name: my-app # because my-app is not listed in the inject annotation above this container will not be injected with secrets
```

          image: my-image
          ports:
            - containerPort: 5000
          command: ["npm"]
          args: ["start"]
          env:
          - name: DB_USERNAME
            value: op://my-vault/my-item/sql/username
          - name: DB_PASSWORD
            value: op://my-vault/my-item/sql/password
```

> **Note**: injected secrets are available in the current pod's session only.

Reviewer: Small nit: small rephrasing to highlight the 'only' part:
```suggestion
**Note**: injected secrets are available *only* in the current pod's session.
```

> In the example above the `app-example1` container will have injected `DB_USERNAME` and `DB_PASSWORD` values.
> But if you try to access them in a new session (for example using `kubectl exec`) it would return 1password item path (aka `op://my-vault/my-item/sql/password`).

Reviewer: We should explain a bit better based on the example what happens. Also, we should include the way of accessing the secrets if another session is made.

I've provided a suggestion in this comment, but feel free to adjust it if you find a better phrasing.
```suggestion
In the example above the `app-example1` container will have injected the `DB_USERNAME` and `DB_PASSWORD` values in the session executed by the command `npm start`.
If you want to access them in a new session (for example using `kubectl exec`) you should append `op run --` to the command executed in the container's new session.

Another alternative to have the secrets available in all container's sessions is by using the [1Password Kubernetes Operator](https://github.com/1password/onepassword-operator).
```

Author: That sounds good to me. Thank you!
